cns/restserver/v2/server.go

```diff
@@ -39,6 +40,9 @@ func (s Server) Start(log *zap.Logger, addr string) {
 	e.GET(cns.V2Prefix+cns.DeleteHostNCApipaEndpointPath, echo.WrapHandler(http.HandlerFunc(s.DeleteHostNCApipaEndpoint)))
 
 	if err := e.Start(addr); err != nil {
-		log.Error("failed to run server", zap.Error(err))
+		log.Error("failed to run echo server", zap.Error(err))
```

Comment:
we should let whoever invoke `Start` to log the error returned instead, and if they already did, then this log would be redundant. Subsequently, we don't need to pass in `*zap.Logger` in the args since the only thing we are logging now is the err